terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -1,21 +1,41 @@
 #!/bin/bash
 # Attach EBS volume to instance
 echo "[$(date '+%H:%M:%S %d-%m-%Y')] installing dependencies for volume attaching"
-sudo yum install -y aws-cli wget
+sudo yum install -y aws-cli wget jq
```

Comment:
can we not add `jq` it is another dependancy to manage, instead use the --output text and POSIX tools (awk/sed). Reason being `jq` has had CVEs in the past which have taken along time to get fixed, I would not be surprised if the version being installed here had open CVEs against it

Reply:
We could not use jq and utilize the AWS cli, I got not issue with that. Everything has CVEs attached to it. Just make sure you stop people from getting to them :D 

We need to have a discussion about vuln management at some point.

Reply:
I have made the change and removed this from the script

Reply:
It's still being installed in line 4 though

Reply:
good spot, thanks, have removed